NEWS.md

```diff
@@ -1,3 +1,26 @@
+# bbr 1.2.1
+
+## Docs
+
+* New functions from the 1.2.0 release are now mentioned in the docs. (#440)
+
+## Changes
+
+* `use_bbi()` no longer depends on external tools such as `wget` and `tar`, hopefully improving its reliability across different systems. (#448)
+
+## Bug fixes
+
+* `use_bbi()` printed the wrong current release when an older version was requested. (#144)
+
+* `use_bbi()` and `bbi_version()` crashed if the bbi executable path contained spaces. (#409)
+
+* `model_summary()` failed with an unclear error message when a caller accidentally placed two `.lst` files in the model directory. (#449)
```

Comment:
"two" - doh, that's misleadingly specific.  Of course, three would have the same issue.  Will reword.